assets/js/data/cart/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { registerStore } from '@wordpress/data';
+import { controls as dataControls } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from './constants';
+import * as selectors from './selectors';
+import * as actions from './actions';
+import * as resolvers from './resolvers';
+import reducer from './reducers';
+import { controls } from '../collections/controls';
```

Comment:
I don't see `apiFetchWithHeaders` used anywhere by the cart store so you don't need to include these in the cart store registration.